### ISSUE TITLE
Switch anthy to maintained fork

### ISSUE THREAD

**Issue by jtojnar** [NixOS/nixpkgs#117562]
anthy has been long unmaintained but the ibus maintainer Takao Fujiwara forked it and maintains it according to https://gitlab.gnome.org/GNOME/gnome-desktop/-/issues/181. It would be nice to switch the package to that:

https://github.com/fujiwarat/anthy-unicode

cc @ericsagnes

**Comment by Aleksanaa** [comment]
Still needing this or not?

**Comment by jtojnar** [comment]
Yes, looks like the unmaintained version is still used: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/inputmethods/anthy/default.nix
